pkg/api/v1/status/team.go

```diff
@@ -23,6 +23,17 @@ func AtlasTeamSetProjects(projects []TeamProject) AtlasTeamStatusOption {
 	}
 }
 
+func AtlasTeamUnsetProject(projectID string) AtlasTeamStatusOption {
+	return func(s *TeamStatus) {
+		for i := 0; i < len(s.Projects); i++ {
+			if s.Projects[i].ID == projectID {
+				s.Projects = append(s.Projects[:i], s.Projects[i+1:]...)
```

Comment:
nit: add a small godoc that this is removing an element from `s.Project` at index `i` preserving order of the remaining elements. This golang idiom might not be obvious to a future reader.